docs/develop/scripting-fundamentals.md

```diff
@@ -111,6 +111,32 @@ Running this script creates the following data in the current worksheet:
 
 :::image type="content" source="../images/range-sample.png" alt-text="A worksheet containing a sales record consisting of value rows, a formula column, and formatted headers":::
 
+### The types of Range values
+
+Each cell has value. This value is the underlying value entered into the cell, which may be different from the text displayed in Excel. For example, you might see "5/2/2021" displayed in the cell as a date, but the actual value is 44318. This display can be changed with the number format, but the actual value and type in the cell only changes when a new value is set.
+
+When you are using the cell value, it's important to tell TypeScript what value you are expecting to get from a cell or range. A cell contains one of the following types: `string`, `number`, `boolean`. In order for your script to treat the returned values as one of those types, you must declare the type.
```

Comment:
[Style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28754) includes conjunctions in lists after a colon (ie: "Los Angeles, Munich, and Tokyo" on the linked page). 
```suggestion
When you are using the cell value, it's important to tell TypeScript what value you are expecting to get from a cell or range. A cell contains one of the following types: `string`, `number`, or `boolean`. In order for your script to treat the returned values as one of those types, you must declare the type.
```